File: vel_pkg/scripts/vel_node.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
# vim:fenc=utf-8
import rospy
from geometry_msgs.msg import Twist

if __name__ == "__main__":
    rospy.init_node("vel_node")
    pub = rospy.Publisher("cmd_vel", Twist, queue_size=10)
    vel_msg = Twist()
    vel_msg.linear.x = 0
    vel_msg.angular.z = 1
    rate = rospy.Rate(30)
    while not rospy.is_shutdown():
        pub.publish(vel_msg)
        rate.sleep()

File: 6-robot/vel_pkg/src/vel_node.cpp
#include <ros/ros.h>
#include <geometry_msgs/Twist.h>

int main(int argc, char *argv[])
{
    ros::init(argc, argv, "vel_node");
    ros::NodeHandle nh;
    ros::Publisher pub = nh.advertise<geometry_msgs::Twist>("/cmd_vel", 10);
    geometry_msgs::Twist vel_msg;
    vel_msg.linear.x = 0.1;
    ros::Rate loop_rate(10);
    while (ros::ok())
    {
        pub.publish(vel_msg);
        loop_rate.sleep();
    }
    return 0;
}